src/controllers/AnimeController.js

```diff
@@ -0,0 +1,11 @@
+animetracker.controller('AnimeController', ['$scope', 'apiservice', '$routeParams', function ($scope, apiservice, $routeParams) {
+    'use strict';
+    var animeId = $routeParams.id;
+    
+    apiservice.getAnime(animeId).then(
+    function(payLoad){
```

Comment:
this payload? shouldnt this be response? please name variable according to the context